pubspec.yaml

```diff
@@ -3,17 +3,19 @@ description: An example app using the angular_components package.
 environment:
   sdk: '>=2.0.0-dev.3.0 <2.0.0'
 dependencies:
-  angular: 5.0.0-alpha+2
-  angular_components: 0.9.0-alpha+2
-  angular_forms: 1.0.1-alpha+2
+  angular: 5.0.0-alpha+4
+  angular_components: 0.9.0-alpha+3
+  angular_forms: 1.0.1-alpha+4
   browser: ^0.10.0
   dart_to_js_script_rewriter: ^1.0.1
-  sass_builder: ^1.1.0
+  sass_builder: 1.1.2
```

Comment:
Might you have meant `^1.1.2` or did you want to pin this specifically to version 1.1.2?

Reply:
Good catch. No I didn't mean to lock it down.